frontend/src/theme/custom.scss

```diff
@@ -90,13 +92,13 @@ ion-item.item-has-focus {
 */
 
 .main-container {
-  width: 100%;
```

Comment:
Why have you removed this line?

Reply:
Maybe a copy/paste error, added this line again ;)